exe/main.cc

```diff
@@ -55,12 +55,14 @@ int main(int ac, char** av) {
   --ac;
   ++av;
 
+  int return_value = 0;
```

Comment:
```suggestion
  auto return_value = 0;
```

AAA style (see https://github.com/motis-project/motis/blob/master/docs/STYLE.md)